### ISSUE TITLE
plans for batch actions

### ISSUE THREAD

**Issue by rall** [marmelab/ng-admin#289]
Are there any plans to allow batch updates and deletes?

**Comment by fzaninotto** [comment]
A few random thoughts.

This is one of the less standardized parts of REST. What do you believe is the common REST API for batch update and delete?

Since it's not standardized, the easy way to implement it in ng-admin would be to issue n update or delete requests to the API - DDOSing it, sort of.

So it's easy to do in a bad way, hard to do the right way, and probably hard to implement support on the server-side.

**Comment by fzaninotto** [comment]
As a side note, plans for the next major release can be read at the bottom of [the 0.5 announcement blog post](http://marmelab.com/blog/2015/02/04/ng-admin-0.5.html).

**Comment by rall** [comment]
ah, thanks for the link. I hadn't seen that. 

For the batch operations, my api supports requests like this:

```
1. PATCH /buttons?attribute=foo { "attribute": "bar" }
2. DELETE /buttons?attribute=qux
```

1. update all 'buttons' resources where attribute="foo", 
2. delete all 'buttons' resources where attribute = "qux"

so I was hoping to be able to send requests like that from ng-admin. My api might be an outlier in the world of Rest on this point, so no big deal if you don't want to support these types of request :-)

**Comment by fzaninotto** [comment]
We'll do that in the future, our customers need it. The hard part is to understand the good practices on the backend side to support that. 

Feel free to implement it for your own needs and send us a PR to see if that can bootstrap the job.

**Comment by fzaninotto** [comment]
Implemented in #374
